azurerm/internal/services/cosmos/common/schema.go

```diff
@@ -89,6 +89,37 @@ func CosmosDbIndexingPolicySchema() *schema.Schema {
 						},
 					},
 				},
+				"composite_index": {
+					Type:     schema.TypeList,
+					Optional: true,
+					Elem: &schema.Resource{
+						Schema: map[string]*schema.Schema{
+							"index": {
+								Type:     schema.TypeList,
+								MinItems: 1,
+								Required: true,
+								Elem: &schema.Resource{
+									Schema: map[string]*schema.Schema{
+										"path": {
+											Type:         schema.TypeString,
+											Required:     true,
+											ValidateFunc: validation.StringIsNotEmpty,
+										},
+										"order": {
+											Type:     schema.TypeString,
+											Required: true,
+											ValidateFunc: validation.StringInSlice(
+												[]string{
+													"ascending",
+													"descending",
```

Comment:
there's constants for these in the Azure SDK:

```
// CompositePathSortOrder enumerates the values for composite path sort order.
type CompositePathSortOrder string

const (
	// Ascending ...
	Ascending CompositePathSortOrder = "Ascending"
	// Descending ...
	Descending CompositePathSortOrder = "Descending"
)
```

so can we make these:

```suggestion
													string(documentdb.Ascending),
													string(documentdb.Descending),
```